setup.py

```diff
@@ -71,6 +71,8 @@
     "bump2version",
     "docsig",
     "no_implicit_optional",
+    "seaborn",
+    "plotly",
```

Comment:
I suppose seaborn and plotly should be in `requirements` variable instead of `dev_requirements`?

Reply:
+1

Reply:
+1